.github/workflows/continuous-integration.yml

```diff
@@ -19,64 +19,28 @@ jobs:
   build:
     name: Build and Run Unit Tests
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [12.x]
     steps:
       - name: Checkout Package
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Using Node.js ${{ matrix.node-version }}
+      - name: Using Node.js 12.x
         uses: actions/setup-node@v1
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: 12.x
       - name: Clean Install
         run: npm ci
       - name: Build
         run: npm run build:release
 
-  check_integ_test_required:
-    name: Check if Integ Tests are Required
-    needs: build
-    continue-on-error: true
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [12.x]
-    outputs:
-      integration_test_required: ${{ steps.integration_test_required.outputs.needs_integration_test }}
-
-    steps:
-      - name: Checkout Package
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - name: Setup Node.js - ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
-        with:
-          node-version: ${{ matrix.node-version }}
-      - name: Check if Integ Tests need to Run
-        run: node script/needs-integration-test.js
-      - name: Set integration test requirement to true if necessary
-        id: integration_test_required
-        if: ${{ success() }}
-        run: echo '::set-output name=needs_integration_test::true'
-
   integ-audio:
     name: Audio Integration Test
-    needs: check_integ_test_required
-    if: ${{needs.check_integ_test_required.outputs.integration_test_required}}
+    needs: build
     runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [12.x]
-
     steps:
       - name: Create a Job ID
         id: create-job-id
-        uses: filipstefansson/uuid-action@v1
+        uses: filipstefansson/uuid-action@ce29ebbb0981ac2448c2e406e848bfaa30ddf04c
```

Comment:
Any reason for the specific commit? It would be good if we document it somehow.

Reply:
That is how it is in the master branch https://github.com/aws/amazon-chime-sdk-js/blob/master/.github/workflows/continuous-integration.yml#L48 so I just copied it over. @michhyun1 can provide more details :D